src/bobtail/bobtail.cpp

```diff
@@ -107,3 +123,13 @@ unsigned int weakPOWfromPOW(unsigned int nBits) {
 
     return a.GetCompact();
 }
+
+double GetKOSThreshold(arith_uint256 target, uint8_t k)
```

Comment:
is it possible to change the math to ints? and then maybe move the decimal after?

Reply:
I don't think so. I'm not aware of an integer type that won't overflow.